data/data/azure/vnet/internal-lb.tf

```diff
@@ -0,0 +1,68 @@
+resource "azurerm_lb" "internal" {
+  sku                 = "Standard"
+  name                = "${var.cluster_id}-ilb"
+  resource_group_name = "${var.resource_group_name}"
+  location            = "${var.region}"
+
+  frontend_ip_configuration {
+    name      = "internal"
+    subnet_id = "${azurerm_subnet.master_subnet.id}"
+  }
+}
+
+resource "azurerm_lb_backend_address_pool" "master_ilb_pool" {
+  resource_group_name = "${var.resource_group_name}"
+  loadbalancer_id     = "${azurerm_lb.internal.id}"
+  name                = "${var.cluster_id}-ilb-master"
+}
+
+resource "azurerm_lb_rule" "internal_lb_rule_api_internal" {
+  name                           = "api-internal"
+  resource_group_name            = "${var.resource_group_name}"
+  protocol                       = "Tcp"
+  backend_address_pool_id        = "${azurerm_lb_backend_address_pool.master_ilb_pool.id}"
+  loadbalancer_id                = "${azurerm_lb.internal.id}"
+  frontend_port                  = 6443
+  backend_port                   = 6443
+  frontend_ip_configuration_name = "internal"
+  enable_floating_ip             = false
+  idle_timeout_in_minutes        = 4
+  load_distribution              = "Default"
+  probe_id                       = "${azurerm_lb_probe.internal_lb_probe_api_internal.id}"
+}
+
+resource "azurerm_lb_rule" "internal_lb_rule_sint" {
+  name                           = "${var.cluster_id}-sint"
+  resource_group_name            = "${var.resource_group_name}"
+  protocol                       = "Tcp"
+  backend_address_pool_id        = "${azurerm_lb_backend_address_pool.master_ilb_pool.id}"
+  loadbalancer_id                = "${azurerm_lb.internal.id}"
+  frontend_port                  = 22623
+  backend_port                   = 22623
+  frontend_ip_configuration_name = "internal"
+  enable_floating_ip             = false
+  idle_timeout_in_minutes        = 4
+  load_distribution              = "Default"
+  probe_id                       = "${azurerm_lb_probe.internal_lb_probe_sint.id}"
+}
+
+resource "azurerm_lb_probe" "internal_lb_probe_sint" {
+  name                = "sint-probe"
+  resource_group_name = "${var.resource_group_name}"
+  interval_in_seconds = 10
+  number_of_probes    = 3
+  loadbalancer_id     = "${azurerm_lb.internal.id}"
+  port                = 22623
+  request_path        = "/healthz"
+  protocol            = "Https"
+}
+
+resource "azurerm_lb_probe" "internal_lb_probe_api_internal" {
```

Comment:
https://github.com/openshift/installer/blob/655fdc7818e3fb888d5878f83aaaceb490d86e02/data/data/aws/vpc/master-elb.tf#L43-L49

Reply:
fixed in d079ea459765d52f918fb0962636b1a05c2286e9

Reply:
The more important change I was requesting was the health check type.
it needs to be a HTTPS `/readyz`

Reply:
fixed the probes. they were all wrong, but that was a change that came up after my first stab at it, and I didn't catch it. https://github.com/openshift/installer/commit/82ae2363e4ba8fce396e2e2c25109eef6b6a941d